examples/README.pod

```diff
@@ -187,3 +187,13 @@
     my $status = HTTP::Status(500);
     Response.new(:$status).write($exception.payload || 'Internal server error.'); # In this case, when we hit an X::AdHoc lets return a 500 with the payload of the exception
   });
+
+=head4 Static files
+
+=for code
```

Comment:
Are you sure this is how it works? Either [the docs](https://docs.raku.org/language/pod#Paragraph_blocks) are wrong, or this will be an empty code block.

Reply:
This is pod not pod6, github doesn't show pod6 README's very nicely so I just use pod.